README.md

```diff
@@ -24,7 +24,8 @@ Documentation for the project can be found [here](https://metoffice.github.io/Ve
 
 The code has been tested with the following compilers:
 - GCC versions: 10.2.0, 11.2.0
-- Clang versions: 12.0 
+- Clang versions: 12.0
+- Cray CCE 15.0.0
```

Comment:
The CI compilers have also been updated so this should be, GCC 12, 13, and 14, Clang (LLVM really) 18.0 and CCE 15.0.0